app/experimenter/experiments/api/v5/types.py

```diff
@@ -185,10 +185,10 @@ def resolve_can_review(self, info):
         return self.can_review(info.context.user)
 
     def resolve_review_request(self, info):
-        return self.changes.latest_review_request()
+        return self.review_request
 
     def resolve_rejection(self, info):
-        return self.changes.latest_rejection()
+        return self.rejection
 
     def resolve_timeout(self, info):
-        return self.changes.latest_timeout()
+        return self.timeout
```

Comment:
Interesting, I thought there was some magic that could infer `resolve_*` when no additional work/params/calls were involved to retrieve a property of the same name

Reply:
Oh hey! Yeah, you're right, I just kind of tweaked my way to this code change but forgot they could just be removed